pom.xml

```diff
@@ -1276,7 +1276,7 @@
             <configLocation>beam/checkstyle.xml</configLocation>
             <suppressionsLocation>beam/suppressions.xml</suppressionsLocation>
             <consoleOutput>true</consoleOutput>
-            <failOnViolation>true</failOnViolation>
+            <failOnViolation>false</failOnViolation>
```

Comment:
<!--new_thread; commit:3bacc3e6099718bbcb672ab738ad607204fa8487; resolved:0-->
Can you isolate this change to the mapreduce runner?